configure.cmake

```diff
@@ -458,6 +477,12 @@ IF(HAVE_SYS_EVENT_H)
 CHECK_FUNCTION_EXISTS (kqueue HAVE_KQUEUE)
 ENDIF()
 
+IF(WITH_SYSTEMD)
+CHECK_FUNCTION_EXISTS (sd_listen_fds HAVE_SYSTEMD_SD_DAEMON_H)
+CHECK_FUNCTION_EXISTS (sd_notify HAVE_SYSTEMD_SD_DAEMON_H)
+CHECK_FUNCTION_EXISTS (sd_notifyf HAVE_SYSTEMD_SD_DAEMON_H)
+ENDIF()
```

Comment:
1. same
2. I'd put all these checks under one IF(), not three.
3. you've copy-pasted these CHECK_FUNCTION_EXISTS and forgot to change the name of the symbol from your previous CHECK_INCLUDE_FILES :)
4. you have five checks: for a library, for a header, for three functions. It's not clear what should happen
if only subset of these checks is satisfied. I'd probably made it impossible. like if not all five
checks are ok - don't use systemd at all.